src/hardware/mixer.cpp

```diff
@@ -112,6 +111,8 @@ struct mixer_t {
 
 	SDL_AudioDeviceID sdldevice = 0;
 	bool nosound = false;
+
+	mixer_t() : resample_temp(MIXER_BUFSIZE * 2), resample_out(MIXER_BUFSIZE * 2) {}
```

Comment:
Maybe these full-up sizings aren't needed now?
Both vectors will growing themselves as-needed in AddSamples (I think)?

Reply:
Agreed, removing.